doc/subsystems/usb/usb.rst

```diff
@@ -1,5 +1,13 @@
+USB device stack
+################
+
+The USB device stack is split into three layers:
```

Comment:
Indenting and line breaks are significant in ReST format:

    The USB device stack is split into three layers, as explained in sections below:

    * USB device controller drivers (hardware dependent)
    * USB device core drivers (hardware independent)
    * USB device class drivers (hardware independent)

Also, these three items are almost the same as the titles of sections below.  If they're intended to be the same, you should make them the same (e.g., "USB device core layer" is the section title below).  You can also make these links to the section headings using the exact title as the link target:

    * `USB device controller drivers`_ (hardware dependent)

Reply:
I have addressed these comments